bin/node-gyp.js

```diff
@@ -25,14 +25,17 @@ prog.parseArgv(process.argv)
 prog.devDir = prog.opts.devdir
 
 var homeDir = osenv.home()
-if (prog.devDir) {
+if (process.env.LOCALAPPDATA && process.platform === 'win32') {
+  prog.devDir = path.resolve(process.env.LOCALAPPDATA, 'nodejs', 'node-gyp')
+} else if (prog.devDir) {
```

Comment:
`prog.devDev` should still be the first one, so that the command line parameter `--devdir` has effect.
```
if (prog.devDir) {
  ...
} else if (process.env.LOCALAPPDATA && process.platform === 'win32') {
  ...
```